packages/simplebar/src/simplebar.js

```diff
@@ -197,14 +203,14 @@ export default class SimpleBar {
       );
     } else {
       // Prepare DOM
-      this.wrapperEl = document.createElement('div');
-      this.contentWrapperEl = document.createElement('div');
-      this.offsetEl = document.createElement('div');
-      this.maskEl = document.createElement('div');
-      this.contentEl = document.createElement('div');
-      this.placeholderEl = document.createElement('div');
-      this.heightAutoObserverWrapperEl = document.createElement('div');
-      this.heightAutoObserverEl = document.createElement('div');
+      this.wrapperEl = elDocument.createElement('div');
```

Comment:
Is it needed to use the helper for this? It's using `document.createElement`, it shouldn't matter much in what document it's being created no?

Reply:
I thought at first that the `ownerDocument` of an element would be set to the document that created it, so I thought doing this would be nice so that adjacent elements in the DOM wouldn't have different documents. Turns out, I'm not entirely sure how, that even if elements created by different documents get mixed together their `ownerDocument` will point to the one that actually contains them.